src/schema/v1/sale_artwork.ts

```diff
@@ -370,6 +371,31 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return saleLoader(sale_id)
         },
       },
+      calculated_cost: {
```

Comment:
The v1 schema has a mix between camelCase and snake_case field names. I think we should stick to the camelCase preference for new additions and it'll also be easier on the eventual V2 upgrade.

Reply:
@alloy @zephraph Should we use camel case in v1? What do you think?

Reply:
If it's new, yes.

Reply:
Aye, all new fields should be camelCase, regardless of schema version. We already started that before v2.

Reply:
Awesome, done: https://github.com/artsy/metaphysics/pull/2004/commits/656c439fb50035dde2355b8ac7ccb5c745d664f7